entries/QUnit.module.xml

```diff
@@ -15,16 +15,28 @@
 				<desc>Runs after each test</desc>
 			</property>
 		</argument>
+		<argument name="contained_suite" optional="true">
+			<desc>A function with contained tests</desc>
+			<type name="Function">
+				<argument name="test" type="Test"></argument>
+			</type>
+		</argument>
 	</signature>
 	<desc>
 		Group related tests under a single label.
 	</desc>
 	<longdesc>
 		<p>
-			All tests that occur after a call to <code>QUnit.module()</code> will be grouped into that module, until other <code>QUnit.module()</code> is called. The test names will all be preceded by the module name in the test results. You can then use that module name to select tests to run.
+			All tests inside a contained suite function will be grouped into that module. The test names will all be preceded by the module name in the test results. If <code>QUnit.module</code> is called without a contained suite argument, the module name will be applied to the consecutive tests calls until another module is defined in the same way.
```

Comment:
I think the final clause (after "If <code>QUnit.module</code> is called without a contained suite argument") should reference test definitions rather than test calls and grouping rather than name prefixing, e.g.:
> All subsequently defined tests will be grouped into the module until another module is defined.